server/views/pages/recallList.njk

```diff
@@ -33,11 +33,13 @@
                                 <td class="govuk-table__cell" data-qa="recallDate">{{ recall.offender.recallDate }}</td>
                                 <td class="govuk-table__cell" data-qa="dueDate">{{ recall.offender.dueDate }}</td>
                                 <td class="govuk-table__cell">
-                                    {% if recall.status == 'RECALL_NOTIFICATION_ISSUED' %}
+                                    {% if recall.status == 'BOOKED_ON' %}
+                                        <div><a href='/persons/{{recall.offender.nomsNumber}}/recalls/{{recall.recallId}}/assess' data-qa='assess-recall-{{ recall.recallId }}'>Assess recall</a></div>
+                                    {% elif recall.status == 'RECALL_NOTIFICATION_ISSUED' %}
```

Comment:
is 'elif' valid?

Reply:
yes! my earlier failures were being caused because I had `else if` and it was failing to run!